lib/irc.js

```diff
@@ -479,24 +479,17 @@ function Client(server, nick, opt) {
                 }
                 break;
             case 'KICK':
-                // channel, who, by, reason
-                self.emit('kick', message.args[0], message.args[1], message.nick, message.args[2], message);
-                self.emit('kick' + message.args[0], message.args[1], message.nick, message.args[2], message);
```

Comment:
In the super switch/case there seem to be redundant emits to the kick listener,  line 483, 484, & 486. 

I'm not sure of the significance of appending message.args[0] to the first parameter or sending as its own parameter, but most other mode/commands are at bottom of the block not at the top. 

Btw - apologies for spamming this PR, I'm trying to learn git and i didn't know how to rebase, but i meant to make a separate PR for kick clean up as i only wanted to submit this small commit: 

https://github.com/martynsmith/node-irc/commit/96a4bdfdd7094da61957d1181a0ea6fc7f14f3ea